modules/installation-and-upgrade/pages/install-vm.adoc

```diff
@@ -46,19 +43,150 @@ Find the {minimalsles} image at https://download.suse.com/.
 For more information on {sle} Virtualization Guide, see https://documentation.suse.com/sles/15-SP4/html/SLES-all/book-virtualization.html.
 ====
 
+[IMPORTANT]
+====
+{productname} VM image does not setup [systemitem]``root`` or any other user account. User or [systemitem]``root`` authentications need to be setup during first boot. This can be done using [systemitem]``Ignition`` or [systemitem]``Cloud-Init`` methods.
+====
+
+[[sumavm.ignition]]
+== {productname} basic configuration using [systemitem]``Ignition``
+
+[systemitem]``Ignition`` is a provisioning tool that enables you to configure a system according to your specification on the first boot.
+When the system is booted for the first time, [systemitem]``Ignition`` is loaded as part of an initramfs and searches for a configuration file within a specific directory (on a USB flash disk, or you can provide a URL).
+
+[systemitem]``Ignition`` uses a configuration file in the JSON format. The file is called [path]``config.ign``.
+
+The [path]``config.ign`` is a JSON configuration file that provides prescriptions for Ignition. You can either create the file manually in JSON, or you can use the Fuel Ignition tool (see https://ignite.opensuse.org/) to generate a basic set of prescriptions. Bear in mind that the Fuel Ignition tool does not provide a full set of options, so you might have to modify the file manually.
+
+When installing, the configuration file [path]``config.ign`` must reside in the [path]``ignition`` subdirectory on the configuration media labeled [systemitem]``ignition``. The directory structure must look as follows:
+
+```
+<root directory>
+└── ignition
+    └── config.ign
+```
+
+If you intend to configure a QEMU/KVM virtual machine, provide the path to the [path]``config.ign`` as an attribute of the qemu command. For example:
+
+```
+-fw_cfg name=opt/com.coreos/config,file=PATH_TO_config.ign
+```
+
+The config.ign contains various data types: objects, strings, integers, Booleans and lists of objects. For a complete specification, refer to Ignition specification v3.3.0 (https://coreos.github.io/ignition/configuration-v3_3/).
+
+
+=== Setup root password using Ignition
+
+The [systemitem]``passwd`` attribute is used to add users. If you intend to log in to your system, create root and set the root's password and/or add the SSH key to the [systemitem]``Ignition`` configuration. You need to hash the root password, for example, by using the openssl command:
+
+```
+openssl passwd -6
+```
+
+The command creates a hash of the password you chose. Use this hash as the value of the [systemitem]``password_hash`` attribute.
+
+The users attribute must contain at least one [systemitem]``name`` attribute. [systemitem]``ssh_authorized_keys`` is a list of ssh keys for the user. 
+
+Prepare required configuration using following commands:
+
+```
+$ mkdir -p root/ignition
+$ cat << EOF > root/ignition/config.ign
```

Comment:
This command mangles the hash in the terminal and in the end you end up with something completely different which is invalid. 
```suggestion
Create a file named root/ignition/config.ign with the following content:
```